helmfile.d/15-ory.yaml

```diff
@@ -0,0 +1,85 @@
+bases:
+- ../environments.yaml
+
+---
+
+releases:
+  - name: kratos
+    chart: ory/kratos
```

Comment:
Please mirror the chart in `external` directory of radar-helm-charts and refer it from there similar to other external charts

Reply:
I copied the Ory helm charts the radar-helm-charts repo. The PR is [here](https://github.com/RADAR-base/radar-helm-charts/pull/202). I updated the chart references in _15-ory.html_ accordingly.

Reply:
Thanks, please also do a quick check if kratos can be successfully installed from our mirror

Reply:
@mpgxvii Did you ever install kratos using the latest radar-kubernetes and helm chart?